docs/helpers/configenvextractor.go

```diff
@@ -63,7 +63,7 @@ func generateIntermediateCode(templatePath string, intermediateCodePath string,
 
 func runIntermediateCode(intermediateCodePath string) {
 	fmt.Println("Running intermediate go code for " + intermediateCodePath)
-	defaultPath := "~/.ocis"
```

Comment:
Won't this result in /etc/ocis/config for the default config path and /etc/ocis/ for the datapath? That sounds wrong.

Reply:
Uhm you are right....
will fix

Reply:
Config path is adjusted later. This is just the base path
```
os.Setenv("OCIS_CONFIG_DIR", path.Join(defaultPath, "config"))
```

Reply:
The defaultPath would be `~/.ocis` resulting in `~/.ocis/config` for the config path.
On a container it the config path would be plain `/etc/ocis` and the respective datapath `/var/lib/ocis`.

Reply:
@rhafer should be fixed now

Reply:
BTW, for our release build we set the paths via build flags. Maybe you can utilize that here as well?

```
-X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseDataPathType=path" -X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseDataPathValue=/var/lib/ocis"
-X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseConfigPathType=path" -X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseConfigPathValue=/etc/ocis"
```